core/bones/base.py

```diff
@@ -59,6 +61,25 @@ class MultipleConstraints:  # Used to define constraints on multiple bones
     preventDuplicates: bool = False  # Prevent the same value of being used twice
 
 
+class ThresholdMethods(Enum):
+    Always = 0
+    Until = 1
+    Once = 2
+
+
+@dataclass
+class ThresholdValue:
+    method: ThresholdMethods = ThresholdMethods.Always
+    seconds: int = 0
```

Comment:
Can we please use a `timedelta` instance here? It's easier to write `timedelta(days=2)` instead of `172800` or `2 * 24 * 60 * 60`.